specification/data-resource-semantic-conventions.md

```diff
@@ -70,6 +70,27 @@ Labels defining a compute unit (e.g. Container, Process, Lambda Function).
 | container.image.name | Name of the image the container was built on. | `gcr.io/opentelemetry/operator` |
 | container.image.tag | Container image tag. | `0.1` |
 
+
+### Function as a Service
+
+**type:** `faas`
+
+**Description:** An instance of a function that runs on a platform where underlying hardware concerns, such as provisioning and scaling, have been abstracted away by the platform.
+
+| Label  | Description  | Example  | Required? |
+|---|---|---|--|
+| faas.name | The name of the function being executed. | `my-function` | Yes |
+<<<<<<< HEAD
```

Comment:
you checked in git resolution markers

Reply:
Damn!! IntelliJ made me lose the [ctrl|cmd]+s muscle memory :smile: